src/Shipping_Method/Settings.php

```diff
@@ -1191,4 +1199,26 @@ public function get_enable_logging() {
 	public function is_logging_enabled() {
 		return ( 'yes' === $this->get_enable_logging() );
 	}
+
+	/**
+	 * Return array of shipping methods.
+	 *
+	 * @return array.
+	 */
+	public function get_shipping_methods() {
+		return  wp_list_pluck( WC()->shipping->get_shipping_methods(), 'method_title', 'id' );
+	}
+
+	/**
+	 * Get supported shipping methods from the settings.
+	 *
+	 * @return array.
+	 */
+	public function get_supported_shipping_methods() {
+		$suppoted_shipping_methods = (array) $this->get_option( 'supported_shipping_methods' );
+		// Add PostNL method by default
```

Comment:
```suggestion
		// Add PostNL method by default.
```
Every comment should end with dot (.)